pkg/speaker/bgp/policy.go

```diff
@@ -0,0 +1,116 @@
+package bgp
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	api "github.com/osrg/gobgp/api"
+	"github.com/osrg/gobgp/pkg/server"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/openelb/openelb/pkg/constant"
+	"github.com/openelb/openelb/pkg/speaker/bgp/config"
+	"github.com/openelb/openelb/pkg/speaker/bgp/table"
+)
+
+func (b *Bgp) UpdatePolicy(cm *corev1.ConfigMap) error {
+	policyConf, ok := cm.Data[constant.OpenELBBgpName]
+	if !ok {
+		b.log.Info("error in %s configmap, %s missing", constant.OpenELBBgpName)
```

Comment:
Does this function need to be called by an external package? Should it detect if cm is nil or not?